### PR TITLE
Add documentation for Poke Jobs farming bot

### PR DESCRIPTION
**Pull request by ErwanLegrand**
Documentation for the Poké Jobs farming bot

### CODE REVIEW

File: Wiki/Programs/PokemonSwSh/DateSpam-PokeJobsFarmer.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+Y-Com glitch must be active
```
Comment (Mysticial) [inline Wiki/Programs/PokemonSwSh/DateSpam-PokeJobsFarmer.md:1]: Copy paste artifacts?

File: Wiki/Programs/PokemonSwSh/DateSpam-PokeJobsFarmer.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+Player must be standing in front of PC or whatever it is called these days
+Boxes before the current boxes must be full or the current box must be box #1
+
+# Date Spam - Poke Jobs
+## Program Description
+
+This program farms Poké Jobs in order to make Pokémons gain experience without evolving or learning new moves. Evolution can later be triggered by leveling or by using Rare Candies is level 100 has been reached. Moves can be learned from the Move Reminder.
+
```
Comment (Mysticial) [inline Wiki/Programs/PokemonSwSh/DateSpam-PokeJobsFarmer.md:9]: Can we can image here? lol :)

File: Wiki/Programs/PokemonSwSh/DateSpam-PokeJobsFarmer.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+## Advanced features
+
+### Mash B for this long upon completion of Poké Job:
+
+   > **SETTINGS THIS TO LESS THAN 8 SECONDS WILL BREAK THE PROGRAM**
+
+Increasing this to 9 seconds or more might fix the program in case it is behaving erratically.
+
+
```
Comment (Mysticial) [inline Wiki/Programs/PokemonSwSh/DateSpam-PokeJobsFarmer.md:50]: Don't forget the server link that every page has.

Comment (ErwanLegrand) [reply Wiki/Programs/PokemonSwSh/DateSpam-PokeJobsFarmer.md:50]: The link was missing in the template. I had noticed this and then I forgot. I have made another pull request to add the link to the template.

File: Wiki/Programs/PokemonSwSh/DateSpam-PokeJobsFarmer.md (hunk 1 above)
Comment (ErwanLegrand) [reply Wiki/Programs/PokemonSwSh/DateSpam-PokeJobsFarmer.md]: Exactly!

File: Wiki/Programs/PokemonSwSh/DateSpam-PokeJobsFarmer.md (hunk 2 above)
Comment (ErwanLegrand) [reply Wiki/Programs/PokemonSwSh/DateSpam-PokeJobsFarmer.md:5]: Sure!

**Comment by ErwanLegrand** [comment]
I have rebased and I have made the required changes.

**Pull request merged**